install.yaml

```diff
@@ -4,7 +4,6 @@ name: adminer
 project_files:
 - docker-compose.adminer.yaml
 - docker-compose.adminer_norouter.yaml
-- adminer
```

Comment:
We need to cleanup old files. Please add this at the end of `pre_install_actions` in this file:

```bash
  - |
    #ddev-nodisplay
    #ddev-description:Removing old adminer files
    has_old_files=false
    for file in "${DDEV_APPROOT}/.ddev/adminer/Dockerfile" "${DDEV_APPROOT}/.ddev/adminer/ddev-login.php"; do
      if [ ! -f "${file}" ]; then
        continue
      fi
      if grep -q '#ddev-generated' "${file}"; then
        rm -f "${file}"
      else
        echo "${file} needs to be removed but has been modified by the user. Please check it and remove it"
        has_old_files=true
      fi
    done
    if [ "${has_old_files}" = true ]; then
      exit 2
    fi
```

Reply:
I added it in [`1bbd8bb` (#30)](https://github.com/ddev/ddev-adminer/pull/30/commits/1bbd8bbe1539b18036194070f6e2d7e49aa58d14)